pyproject.toml

```diff
@@ -52,7 +52,11 @@ looponfailroots = ["src", "tests"]
 [tool.mypy]
 python_version = "3.11"
 mypy_path = ["fixtures/stubs-for-mypy"]
-plugins = ["pydantic.mypy", "mypy_django_plugin.main", "tools.mypy_helpers.plugin"]
+plugins = [
+    "pydantic.mypy",
+    "mypy_django_plugin.main",
+    "tools.mypy_helpers.plugin",
+]
```

Comment:
unrelated to your change and not a helpful reformat imo

Reply:
would be nice to have a TOML formatter in pre-commit as my default toml formatter always makes this change. i've turned off my TOML formatter for future PRs here